BE/OwnershipAndControl/CorporateOwnership.rdf

```diff
@@ -52,22 +56,40 @@
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/AnnotationVocabulary/"/>
-		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/BE/20190201/OwnershipAndControl/CorporateOwnership/"/>
+		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/BE/20200601/OwnershipAndControl/CorporateOwnership/"/>
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/BE/20131101/OwnershipAndControl/CorporateOwnership.rdf version of this ontology was modified per the issue resolutions identified in the FIBO BE 1.0 FTF report.</skos:changeNote>
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/BE/20160201/OwnershipAndControl/CorporateOwnership.rdf version of this ontology was modified per the FIBO 2.0 RFC to reference shareholders&apos; equity in the definition of a shareholder.</skos:changeNote>
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/BE/20180801/OwnershipAndControl/CorporateOwnership.rdf version of this ontology was modified to generalize the definition of beneficial owner rather than limiting it to shareholding and eliminate a duplicate restriction on shareholder.</skos:changeNote>
+		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/BE/20190201/OwnershipAndControl/CorporateOwnership.rdf version of this ontology was modified to modify the inheritance hierarchy for beneficial owner to replace owner with controlling party as one of its parent classes.</skos:changeNote>
 		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Release"/>
 	</owl:Ontology>
 	
 	<owl:Class rdf:about="&fibo-be-oac-cown;BeneficialOwner">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-agr-agr;Beneficiary"/>
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-oac-own;Owner"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-oac-ctl;ControllingParty"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-pty-rl;isPlayedBy"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;controls"/>
+						<owl:someValuesFrom>
+							<owl:Restriction>
+								<owl:onProperty rdf:resource="&fibo-fnd-pty-rl;playsRole"/>
+								<owl:someValuesFrom rdf:resource="&fibo-fnd-oac-own;Asset"/>
+							</owl:Restriction>
+						</owl:someValuesFrom>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
 		<rdfs:label>beneficial owner</rdfs:label>
-		<skos:definition>a party that enjoys the possession and/or benefits of ownership (such as receipt of income) of a property even though its ownership (title) is in the name of another party (called a nominee or registered owner)</skos:definition>
+		<skos:definition>party that enjoys the possession and/or benefits of ownership (such as receipt of income) of a property even though its ownership (title) is in the name of another party (called a nominee or registered owner)</skos:definition>
```

Comment:
"of a property" seems over-specific here.
Likewise requiring it to be an Asset (which apparently must involve future cash inflows) is over-restrictive. 
If beneficial owner must be different from actual owner, should there not be a disjoint restriction?

Reply:
I've loosened the restriction to be 'min 0' with respect to controlling an asset and eliminated a synonym of 'actual owner', which was not correct.  i've also changed "of a property" to "of something".

Note that there are cases where someone is called a beneficial owner that is the business owner of an LLC or partnership, even though that individual is at arms length due to the nature of the legal entity.  So I am reticent to add the disjointness relation, which could cause problems in those cases.